### ISSUE TITLE
removeUserFromRoom doesn't trigger onUserLeft

### ISSUE THREAD

**Issue by anchetaWern** [pusher/chatkit-client-js#159]
I'm creating a chat app which has admin privileges in it. The admin can remove someone from the room. I'm using the `removeUserFromRoom` method (client-side) for that. The scenario I'm testing for is that when the admin and the member is chatting. And the admin removes the member they're currently chatting with in the room. In this case, I was expecting for the `onUserLeft` hook to be triggered the moment the admin calls `removeUserFromRoom`, but it didn't. So when the removed member sends a message, we now get the error: "room membership required".

**Comment by callum-oakley** [comment]
If a user is removed from a room, all the other members of the room will get a `onUserLeft` event, but the user that was removed will get an [onRemovedFromRoom](https://pusher.com/docs/chatkit/reference/javascript#connection-hooks) event. Does that explain what you're seeing?

**Comment by anchetaWern** [comment]
Ah ok. Looks like I missed that event hook. Let me try again.

**Comment by callum-oakley** [comment]
Closed as stale.
